git-gud.1

```diff
@@ -0,0 +1,68 @@
+.TH git-gud 1 "19 Feb 2020" "version 0.0"
+.SH NAME
+git-gud \- get good at git with git-gud
+.SH SYNOPSIS
+.B git gud
+[option1] [option2]
+.SH DESCRIPTION
+.B git-gud
+teaches you how to use Git with fun interactive tutorials.
+Use the following command to create a Git repository and start the first level:
+.B python\ \-m\ gitgud\ start
+.SH OPTIONS
+.TP
+\fBstatus\fP
+Displays current level.
+.TP
+\fBinstructions\fP
+Displays instructions for current level.
+.TP
+\fBreset\fP
+Resets the current level.
+.TP
+\fBtest\fP
+Tests if current level is complete.
+.TP
+\fBload\ next\fP
+Takes you to the next level.
+.TP
+\fBskills\fP
+Displays all skills.
+.TP
+\fBlevels\fP
+Displays all levels in the current skill.
+.TP
+\fBlevels\ <skill_name>\fP
+Displays all levels in the specified skill.
+.TP
+\fBload\ <skill_name>\fP
+Begin the first level of the specified skill.
+.TP
+\fBload\ \-<level_name>\fP
+Begin the specified level of the current skill.
+.TP
+\fBload\ <skill_name>\ <level_name>\fR, \fBload\ <skill_name>\-<level_name>\fP
+Begin the specified level of a skill.
+.TP
+\fBcommit\fP
+Commit an empty file with a unique name.
+.TP
+\fBcommit\ <commit_name>\fP
+Commit an empty file with the specified name.
+.TP
+\fBgoal\fP
+Concisely prints out what is needed to complete the level.
+.TP
+\fBshow\-tree\fP
+Visualizes the current git tree.
+.SH BUGS
+If you believe you have found a bug in \fBgit-gud\fR, please file an issue at this address https://github.com/benthayer/git-gud/issues/
+.SH AUTHOR
+Written by Evan LeClercq (eleclercq@tuta.io).
```

Comment:
Can you add a link to the github page here so people can find all the authors of git gud themselves. Give me explicit credit for starting it, and mention OSAI. It's also fine to give yourself explicit credit for the man page as you've done.

It'd probably make sense to follow the style of the git man page for this section

Reply:
I filed this under #145: "Add CONTRIBUTORS file to repository". Man pages are meant to be local manuals, so having a link in this section is a *faux pas*, which is why I suggested adding a contributing section. 

Also, note that the `AUTHOR` section of a man page is the person who wrote the man page. The `AUTHORS` section is reserved for both the person or persons who authored the man page **and** those who have contributed to the development of the program.

For now I am leaving this; however, upon resolution of issue #145, I will add that file to the man page and update the section to `AUTHORS` to reflect that change.

Reply:
Can I see a source for this?

I found [this](http://man7.org/linux/man-pages/man7/man-pages.7.html), which says the use of `AUTHORS` is strongly discouraged. When I run `man vim` and `man git`, the `AUTHOR` section 

Git man page:
```
AUTHORS
       Git was started by Linus Torvalds, and is currently maintained by Junio C Hamano. Numerous contributions have
       come from the Git mailing list <git@vger.kernel.org[5]>. http://www.openhub.net/p/git/contributors/summary
       gives you a more complete list of contributors.

       If you have a clone of git.git itself, the output of git-shortlog(1) and git-blame(1) can show you the authors
       for specific parts of the project.
```

Vim man page:
```
AUTHOR
       Most of Vim was made by Bram Moolenaar, with a lot of help from others.  See ":help credits" in Vim.
       Vim  is based on Stevie, worked on by: Tim Thompson, Tony Andrews and G.R. (Fred) Walter.  Although hardly any
       of the original code remains.
```

These seem to be similar usage, and given that use of the `AUTHORS` section is "strongly discouraged", I think the git approach of ```If you have a clone of git.git itself, the output of git-shortlog(1) and git-blame(1) can show you the authors for specific parts of the project.``` It also seems like `AUTHOR` and `AUTHORS` are the same section, just renamed depending on the number of authors, so it wouldn't make sense to have both sections.

Also, as this project is based on git, I want to follow their convention. It seems like the git repo contains the equivalent of the CONTRIBUTORS file and using the git log instead makes our lives easier.

So the bottom line would be to have that same quote from the git repository while also citing the other things I said.

I'd like to hear your thoughts.